### PR TITLE
Python Program to Swap Two nibbles

### PR DESCRIPTION
**Pull request by narayan2111**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Python Program to  swap the two nibbles in given byte. For example 100 is be represented as 01100100 in a byte (or 8 bits).
The two nibbles are (0110) and (0100). If we swap the two nibbles, we get 01000110 which is 70 in decimal.

### Checklist

- [Y ] I've read the contribution guidelines.
- [Y ] I've checked the issue list before deciding what to submit.
- [Y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#980

### CODE REVIEW

**Comment by narayan2111** [comment]
kindly review this PR @plazzy99 @iamrajiv @Kajol-Kumari @Abhijit2505 @HarshCasper

**Comment by narayan2111** [comment]
kindly review this PR @plazzy99 @iamrajiv @atarax665 @Kajol-Kumari @Abhijit2505

**Comment by narayan2111** [comment]
kindly review this PR @iamrajiv @plazzy99 @Kajol-Kumari @atarax665 @Abhijit2505

**Review by iamrajiv — APPROVED** [review]

File: Python/other/Swap_two_nibbles.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# python program Swap two nibbles in a byte
```
Comment (atarax665) [inline Python/other/Swap_two_nibbles.py]: Improve the program description.

**Review by atarax665 — APPROVED** [review]
LGTM :+1: , Do suggested change.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Comment by narayan2111** [comment]
Kindly merge this issue, it has 3 pull requests @iamrajiv

**Comment by narayan2111** [comment]
kindly merge this PR @iamrajiv

**Pull request merged**